### PR TITLE
Registration from sumit

### PR DESCRIPTION
**Pull request by sumit121dev**
Please accept my request and thank you so much for this initiative.

### CODE REVIEW

File: content/participants copy/ttjc@sumit121dev/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: dummy student
```
Comment (gkanishk) [inline content/participants copy/ttjc@sumit121dev/index.md]: Please change the title with your **name** and not **dummy student**

Comment (sumit121dev) [reply content/participants copy/ttjc@sumit121dev/index.md]: @gkanishk sir, shall I raise a new PR or just update the relevant file here only?

Comment (sumit121dev) [reply content/participants copy/ttjc@sumit121dev/index.md]: never mind, just saw the changed label! Thank you so much for working this late. Your commitment inspires us!

Comment (gkanishk) [reply content/participants copy/ttjc@sumit121dev/index.md]: Your welcome man!!

Comment (sumit121dev) [reply content/participants copy/ttjc@sumit121dev/index.md]: [Noob question] just to be clear, I don't need to participate again at tomorrow morning, my pr will get accepted and I can happily code in peace?

**Comment by gkanishk** [comment]
Yes you don't need to do that again

**Comment by sumit121dev** [comment]
Well then I'd get back to coding :)

**Pull request merged**